releasenotes/notes/asm-rcm-feature-registration-21d79315c7a0feb7.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    ASM: add remote configuration management feature registration support.
```

```suggestion
    ASM: Enable ability to remotely enable and configure ASM. To enable, check the Python Security page in your account.
```


Bad suggestion, but rough idea.

---

I'll commit:

```
    ASM: Enable ability to remotely activate and configure ASM features. To 
    enable, check the Python Security page in your account. Note that this is a beta
    feature.
```